Bottom: b929ebbb9b800f98eac7f635ec6784ae61b8c908
Top:    836d6b4fdaa7a0ace3c82a2bdeeb85608587989f
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-18 19:52:46 +1300

Refresh of timestamp-stream-result

---

diff --git a/NEWS b/NEWS
index 984dc5d..cf6eb0b 100644
--- a/NEWS
+++ b/NEWS
@@ -27,9 +27,6 @@ Improvements
 * New class ``StreamResult`` which defines the API for the new result type.
   (Robert Collins)
 
-* New support class ``TimestampingStreamResult`` which adds a timestamp to
-  events with no timestamp. (Robert Collins)
-
 * New support class ``CopyStreamResult`` which forwards events onto multiple
   ``StreamResult`` objects (each of which receives all the events).
   (Robert Collins)
@@ -65,6 +62,9 @@ Improvements
   ``TestResult``) calls. This permits using un-migrated result objects with
   new runners / tests. (Robert Collins)
 
+* New support class ``TimestampingStreamResult`` which adds a timestamp to
+  events with no timestamp. (Robert Collins)
+
 * New ``TestCase`` decorator ``DecorateTestCaseResult`` that adapts the
   ``TestResult`` or ``StreamResult`` a case will be run with, for ensuring that
   a particular result object is used even if the runner running the test doesn't
